lib/scanner.js

```diff
@@ -50,6 +51,19 @@ function isOpened () {
   return opened
 }
 
+function hasCamera (mode) {
+  const cameraConf = mode === 'facephoto' ? 'frontFacingCamera' : 'scanner'
+
+  const device = _.get(`${cameraConf}.device`, configuration)
+  const modeConfig = _.get([cameraConf, mode], configuration)
```

Comment:
Shouldn't we check for null device?

Reply:
Good point, the try catch would handle this case but I added a new test for it.